src/se/vidstige/jadb/RemoteFileRecord.java

```diff
@@ -23,12 +23,25 @@ public int getSize() {
     }
 
     @Override
-    public long getLastModified() {
+    public int getLastModified() {
         return lastModified;
     }
 
     @Override
     public boolean isDirectory() {
         return (mode & (1 << 14)) == (1 << 14);
     }
+
+    @Override
+    // This is to explicitly state that although the class has additional fields,
+    // it still uses inherited equals() and hashCode().
+    // In other words, we didn't forget to implement proper equals().
+    public boolean equals(Object o) {
+        return super.equals(o);
+    }
+
+    @Override
+    public int hashCode() {
+        return super.hashCode();
+    }
```

Comment:
This seems odd... :thinking: Can we just have the comment? Perhaps the warning can be ignored somehow?

Reply:
I've removed the whole commit.